tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/clean_smoking_test.py

```diff
@@ -0,0 +1,113 @@
+"""
+Integration test for CleanSmokingPpi
+"""
+
+# Python imports
+import os
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.clean_smoking_ppi import (CleanSmokingPpi,
+                                                          SMOKING_LOOKUP_TABLE)
+from common import JINJA_ENV, OBSERVATION
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+LOAD_QUERY = JINJA_ENV.from_string("""
+    INSERT INTO `{{project_id}}.{{dataset_id}}.observation` (
+        observation_id,
+        person_id,
+        observation_concept_id, 
+        observation_source_concept_id,
+        value_as_concept_id,
+        value_source_concept_id
```

Comment:
You're going to have to load some test data for `observation_date` and `observation_type_concept_id`.  These fields are required in the OMOP schema and the test table schemas are built from our json files.  You can fill them with dummy data, they just have to be in the loaded data.  

You are not required to test their cleaned versions.  Lines 96 - 110 do not have to change.